jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -292,8 +314,8 @@ boolean isRPCServer() {
         }
       }
 
-      if (parent == null || isRPCServer()) {
-        // add tracer tags only to first span in the process
+      // add tracer tags to zipkin first span in process
+      if (zipkinSharedRPCSpan && (preferredParent == null || isRPCServer())) {
```

Comment:
this looks like an overloading of this boolean flag. I think it's better to add tracer tags in the Sender, since it's specific to Zipkin. But if it creates too much complexity, I'm ok to leave it here, maybe add a TODO